ArduCopter/Parameters.cpp

```diff
@@ -963,6 +963,12 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     AP_SUBGROUPINFO(arot, "AROT_", 37, ParametersG2, AC_Autorotation),
 #endif
 
+    // @Param: AUTO_MAX_WP_DIST
+    // @DisplayName: Maximum allowed distance to first waypoint in mode auto
+    // @Description: Maximum allowed distance in meters to first waypoint to allow switching to mode auto. Prevents a fly away. Set param to 0, to disable the feature. 
+    // @User: Standard
```

Comment:
The units need to be specified using @Units: and a @range: must also be specified.

Reply:
@amilcarlucas I wanted to put the range there but I got a little confused on what I should write. Setting '0' disables the features and the range can be anything that `uint32_t` can store. So should I put range to soemthing like 10-10000 (10km)

Reply:
something like:
`@range: 0 20000`
should do it.